src/sponsorableFetcher.ts

```diff
@@ -0,0 +1,88 @@
+//to-do: get sponsorable contributors
```

Comment:
I would favor leaving sponsorables out of this MR entirely, and having this one just list contributors. Your choice, though.